pom.xml

```diff
@@ -30,22 +30,22 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
-            <version>2.8.0</version>
+            <version>2.9.7</version>
```

Comment:
You can make this modular by adding

`<jackson.version>2.9.7</jackson.version>`

in the `<properties></properties>` section above, and referring to it here with

`<version>${jackson.version}</version>`